manim/mobject/types/opengl_image_mobject.py

```diff
@@ -0,0 +1,53 @@
+from typing import Union
```

Comment:
Can you use the new typing style rather than importing these old typehints?

Reply:
Ah, okay, that makes sense